CONTRIBUTING.md

```diff
@@ -47,7 +47,10 @@ Plugins which fall under the following categories will not be implemented or con
 
 7. Sites where the maintainer has requested we not add their site to Streamlink
 
-8. Sites which are unmaintained.
+8. Sites which are unmaintained
+
+9. Sites which don't provide any video streaming content, eg. radio stations
+
```

Comment:
I think it's worth adding a few additional lines such as:
```
10. Sites which create a live stream of previously recorded content, such as TV shows, movies, news broadcasts, etc.

11. Sites which have no way to determine viewership numbers

12. Sites which are static cameras of a physical location

13. Sites which are in beta or undergoing heavy amounts of development and may change rapidly
```

This should cover us for more plugins and keep people from wasting their time and ours.

Reply:
I agree with all of those, but I feel like a list of 13 points makes it a bit too much IMO.

Reply:
Agreed, your changes are good.